docs/topics/benchmarking.rst

```diff
@@ -81,5 +81,5 @@ follow links, any custom spider you write will probably do more stuff which
 results in slower crawl rates. How slower depends on how much your spider does
 and how well it's written.
 
-In the future, more cases will be added to the benchmarking suite to cover
-other common scenarios.
+To use it as a project for more complex Scrapy benchmarking:
+https://github.com/scrapy/scrapy-bench
```

Comment:
What about a shorter “Use scrapy-bench for more complex benchmarking”? (linking “scrapy-bench” to https://github.com/scrapy/scrapy-bench)